loopd/daemon.go

```diff
@@ -72,9 +75,9 @@ func New(config *Config, lisCfg *listenerCfg) *Daemon {
 	}
 }
 
-// Run runs loopd in daemon mode. It will listen for grpc connections,
+// Start starts loopd in daemon mode. It will listen for grpc connections,
 // execute commands and pass back swap status information.
-func (d *Daemon) Run() error {
+func (d *Daemon) Start() error {
```

Comment:
I am not too sure about the commit structure. It feels that I am seeing the lines change multiple times too often. Perhaps some squashing is appropriate. As a reviewer, I moved to reviewing the full diff because of this. And that defeats the purpose of having a commit structure.